vignettes/SEMinR.Rmd

```diff
@@ -479,6 +480,23 @@ confidence_interval(boot_seminr_model = boot_seminr_model,
                    alpha = 0.10)
 ```
 
+### Reporting data descriptive statistics and construct descriptive statistics with `summary(seminr_model)`
+
+The `summary(seminr_model)` function will return four matrices:
+`model_summary <- summary(seminr_model)` returns an object of class `summary.seminr_model` which contains the following four descriptive statistics matrices:
+    + `model_summary$descriptive_stats$item_descriptives` reports the descriptive statistics for items
+    + `model_summary$descriptive_stats$item_correlations` reports the correlation matrix for items
+    + `model_summary$descriptive_stats$construct_descriptives` reports the descriptive statistics for constructs
+    + `model_summary$descriptive_stats$construct_correlations` reports the correlation matrix for constructs
+
+```{r}
+model_summary <- summary(mobi_pls)
+model_summary$descriptive_stats$item_descriptives
```

Comment:
What do you think if try to DRY the names a bit:
`model_summary$descriptives$correlations$items`
`model_summary$descriptives$descriptives$constructs`